agenta-web/src/components/EvaluationTable/SingleModelEvaluationTable.tsx

```diff
@@ -565,6 +575,14 @@ const SingleModelEvaluationTable: React.FC<EvaluationTableProps> = ({
                 />
             </div>
 
+            <SaveTestsetModal
+                setIsTestsetModalOpen={setIsTestsetModalOpen}
+                isTestsetModalOpen={isTestsetModalOpen}
+                appId={appId}
```

Comment:
you can avoid passing in `appId` as the component can directly use `useAppId` hook to get it